tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -872,6 +872,101 @@ TEST(RDFSnapshotMore, RedefinedDSColumn)
    gSystem->Unlink(fname);
 }
 
+// https://github.com/root-project/root/issues/6932
+TEST(RDFSnapshotMore, MissingSizeBranch)
+{
+   const auto inFile = "test_snapshot_missingsizebranch.root";
+   const auto outFile = "test_snapshot_missingsizebranch_out.root";
+
+   // make input tree
+   {
+      TFile f(inFile, "recreate");
```

Comment:
Would it make sense to use in-memory trees or TMemFile for these kind of tests?

Reply:
yes in principle it makes sense, in practice I don't know if it works e.g. with TTreeProcessorMT (which has to re-open the file in each thread). I noted it down to try and convert all our RDF tests to use TMemFiles whenever possible, but I'd address this separately.